data/assets/scene/digitaluniverse/localdwarfs.asset

```diff
@@ -25,17 +25,26 @@ local Object = {
       MinMaxSize = { 7, 20 },
       Unit = "Mpc"
     },
-    Color = { 0.5, 1.0, 0.2 },
-    ColorMap = speck .. "localgroup.cmap",
-    ColorOption = { "association" },
+    Coloring = {
+      FixedColor = { 0.0, 1.0, 0.0 },
+      -- @TODO: This one wasn't actually properly used before the point cloud update.
```

Comment:
Decide if this should be kept around

Reply:
What do you mean?  Keeping the ability to have a fixed color?

Reply:
The color map was not correctly read.  

Although, looking at this in more details there are some problems with the speck file for this dataset that I think we need to discuss...

Reply:
https://github.com/OpenSpace/OpenSpace/issues/2983